azure-pipelines.yml

```diff
@@ -3,6 +3,7 @@ trigger:
   branches:
     include:
     - master
```

Comment:
Why still include master?

Reply:
It's part of the instructions, I'll remove it once the final move is finished.